src/clj/qbits/jet/client/http.clj

```diff
@@ -321,6 +323,14 @@
                                           (.add f (name k) (str v)))
                                         f))))
 
+       (when (seq multipart)
+             (.content request
+                       (let [provider (MultiPartContentProvider.)]
+                            (doseq [[k v] multipart]
+                                   (.addFieldPart provider k (StringContentProvider. v) nil))
```

Comment:
There may be some opportunities to improve the API.  For example, if a string is passed as `v`, then maybe you automatically wrap it in a `StringContentProvider`.  Is that what the caller is doing?  Might be nice to have that here.